### PR TITLE
add SyncMode option to PebbleExtraOptions

### PR DESCRIPTION
**Pull request by magicxyyz**
Resolves NIT-2952

### CODE REVIEW

**Review by diegoximenes — APPROVED** [review]

**Review by tsahee — APPROVED** [review]
LGTM

**Comment by tsahee** [comment]
Need to spend some time thinking what should be default for us

File: ethdb/pebble/pebble.go [hunk 1]
```diff
@@ -238,7 +238,7 @@ func New(file string, cache int, handles int, namespace string, readonly bool, e
 		fn:           file,
 		log:          logger,
 		quitChan:     make(chan chan error),
-		writeOptions: &pebble.WriteOptions{Sync: !ephemeral},
```
Comment (tsahee) [inline ethdb/pebble/pebble.go:241]: I think I'll feel better if our default will stay "on" and we could later think if the logic that goes for geth turning it off also fits us

**Review by tsahee — CHANGES_REQUESTED** [review]

**Review by tsahee — APPROVED** [review]
LGTM

**Pull request merged**